### DISCUSSION TITLE
Tracking

### DISCUSSION THREAD

**Discussion by animafps** [aiming-taxonomy/glossary#9] [Concepts]
Current definition:
`The skill to maintain your crosshair on a target for a long period of time while it moves and can reacquire the same target after it changes direction. Often long time-to-kill targets with a high fire-rate weapon.`

**Comment by deLiGHT-03** [comment]
The ability to read a target's movement, the mouse control to stay on target and the reactivity to respond quickly to changes in direction.  

I think that's the core. There are some sub-strategies to tracking like Preloading but I wouldn't consider it core to it.

**Reply by animafps** [reply]
Reading the movement is part of reading skills and are applied, wouldn't work it into the definition though, and "respond quickly to changes" is way to ambiguous to what the resulting movement should be.
wtf is preloading, been in this community for over 2 years and never heard that term

**Reply by deLiGHT-03** [reply]
Preloading is just shorthand for anticipatory reaction. It's mentally preparing for a movement before it happens. The common example is tracking a target to the right, while mentally waiting for the counter-strafe. Preps some of the calculation in advance and shaves down time.

**Reply by animafps** [reply]
So predicting, yeah its not quite specific enough to tracking to be part of it or necessary effect the definition

**Comment by riddbtw** [comment]
Kovaak definition:

> Player uses a fully-automatic weapon weapon and maintains their crosshair over the target while firing for an extended period of time.

I agree more with Anima's definition, because while tracking is _usually_ on higher-ttk targets with fully automatic guns, that's not always the case.  Sometimes you need to use tracking skills with guns that are more traditionally clicking guns, like certain types of shotguns.
